src/control/cmd/dmg/storage_query.go

```diff
@@ -142,16 +156,33 @@ func (cmd *usageQueryCmd) Execute(_ []string) error {
 		return err
 	}
 
-	var bld strings.Builder
-	if err := pretty.PrintResponseErrors(resp, &bld); err != nil {
+	var outErr strings.Builder
+	if err := pretty.PrintResponseErrors(resp, &outErr); err != nil {
 		return err
 	}
-	if err := pretty.PrintHostStorageUsageMap(resp.HostStorage, &bld); err != nil {
-		return err
+	if outErr.Len() > 0 {
+		cmd.Error(outErr.String())
+	}
+
+	var out, dbg strings.Builder
+	if resp.HostStorage.IsMdOnSsdEnabled() {
+		if err := pretty.PrintHostStorageUsageMapMdOnSsd(resp.HostStorage, &out, &dbg, cmd.ShowUsable); err != nil {
+			cmd.Error(err.Error())
+		}
+	} else {
+		if cmd.ShowUsable {
+			cmd.Notice("--show-usable flag ignored when MD-on-SSD is not enabled")
+		}
+		pretty.PrintHostStorageUsageMap(resp.HostStorage, &out)
+	}
+	if dbg.Len() > 0 {
+		cmd.Debugf("%s", dbg.String())
+	}
+	if out.Len() > 0 {
+		// Infof prints raw string and doesn't try to expand "%"
+		// preserving column formatting in txtfmt table
+		cmd.Infof("%s", out.String())
```

Comment:
```suggestion
		cmd.Info(out.String())
```

Reply:
this is useful because some parsing of escape characters is different if you run it through format